### PR TITLE
 Customizations for UNICEF Angola

### PR DESCRIPTION
**Pull request by hamza-vd**
If the parameter vaccine.overdue.status.hide is set to true then due and over due statuses colors will be white. No indication will be shown for these statuses.

### CODE REVIEW

File: opensrp-child/src/main/java/org/smartregister/child/task/VaccinationAsyncTask.java [hunk 1]
```diff
@@ -69,6 +69,7 @@
     private View childProfileInfoLayout;
     private boolean isLegacyAlerts = ChildLibrary.getInstance().getProperties().hasProperty(ChildAppProperties.KEY.HOME_ALERT_STYLE_LEGACY) && ChildLibrary.getInstance().getProperties().getPropertyBoolean(ChildAppProperties.KEY.HOME_ALERT_STYLE_LEGACY);
     private boolean upcomingLightBlueDisabled = ChildLibrary.getInstance().getProperties().hasProperty(ChildAppProperties.KEY.HOME_ALERT_UPCOMING_BLUE_DISABLED) && ChildLibrary.getInstance().getProperties().getPropertyBoolean(ChildAppProperties.KEY.HOME_ALERT_UPCOMING_BLUE_DISABLED);
+    private boolean hideOverdueVaccine = ChildLibrary.getInstance().getProperties().hasProperty(ChildAppProperties.KEY.VACCINE_OVERDUE_STATUS_HIDE) && ChildLibrary.getInstance().getProperties().getPropertyBoolean(ChildAppProperties.KEY.VACCINE_OVERDUE_STATUS_HIDE);
```
Comment (ellykits) [inline opensrp-child/src/main/java/org/smartregister/child/task/VaccinationAsyncTask.java:72]: Change this to **hideOverdueVaccineStatus**

File: opensrp-child/src/main/java/org/smartregister/child/util/ChildAppProperties.java [hunk 2]
```diff
@@ -41,10 +40,12 @@
 
         public static final String FEATURE_RECURRING_SERVICE_ENABLED = "recurring.services.enabled";
 
+        // vaccine status color
+        public static final String VACCINE_OVERDUE_STATUS_HIDE = "vaccine.overdue.status.hide";
```
Comment (ellykits) [inline opensrp-child/src/main/java/org/smartregister/child/util/ChildAppProperties.java:44]: Rename to **HIDE_OVERDUE_VACCINE_STATUS = "hide.overdue.vaccine.status"**

File: sample/src/main/assets/app.properties [hunk 3]
```diff
@@ -22,3 +22,4 @@ mother.lookup.undo.duration=5000
 widget.datepicker.is.numeric=false
 vaccine.expired.entry.allow=true
 novel.out.of.catchment=true
+vaccine.overdue.status.hide=false
```
Comment (ellykits) [inline sample/src/main/assets/app.properties:25]: Question is this necessary if this is the default behavior? 
Remember to update to this **hide.overdue.vaccine.status**

**Review by ellykits — CHANGES_REQUESTED** [review]

**Comment by ellykits** [comment]
Also remember to update your branch with master and add library version on **gradle.properties** file

File: sample/src/main/assets/app.properties (hunk 3 above)
Comment (hamza-vd) [reply sample/src/main/assets/app.properties:25]: Yes  `vaccine.overdue.status.hide` will be false by default if it isn't declared here. I have declared it just for the reference. Should I remove it?

Comment (ellykits) [reply sample/src/main/assets/app.properties:25]: Remove if not necessary

**Comment by ellykits** [comment]
:+1: Approved

**Review by ellykits — APPROVED** [review]

**Pull request merged**